src/components/Book.js

```diff
@@ -0,0 +1,68 @@
+// Individual Book component
+
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { removeBook } from '../redux/books/booksSlice';
+import { RemoveBookButton } from './Button';
+import '../styles/book.css';
+import progress from '../Assets/progress.png';
+
+const Book = ({
+  id, title, author, category,
+}) => {
+  const dispatch = useDispatch();
+
+  const handleRemoveClick = () => {
+    dispatch(removeBook(id));
+  };
+  return (
+    <div className="book">
+      <div className="book-cover">
+        <div className="book-info">
+          <h6 className="category">
+            {category}
+            Action
+          </h6>
+          <h3 className="title">
+            {title}
+          </h3>
+          <p className="author">
+            {author}
+          </p>
+        </div>
+        <div className="book-actions">
+          <p className="comments">
+            Comments
+          </p>
+          <span className="remove">
+            <RemoveBookButton className="remove-btn" type="button" onClick={handleRemoveClick}>Remove</RemoveBookButton>
+          </span>
+          <p className="edit">
+            Edit
+          </p>
+        </div>
+      </div>
+      <div className="progress">
+        <img src={progress} alt="progress" />
+        <div className="progress-info">
+          <p className="parcent">64%</p>
+          <p className="completed">Completed</p>
+        </div>
```

Comment:
- [ ] As I mentioned above please stick to the Zaplin design but your Completed text colour and font style do not match the Zaplin design so kindly follow the Zaplin design as much as possible :+1: check below :point_down: the comparison between your design and Zaplin design :slightly_smiling_face: 

**Your Design**
![Screenshot from 2023-09-15 17-57-50](https://github.com/samuelkinyanjui/bookstore_add_reducers/assets/45226848/bc19bbe5-0dfc-41e9-b8d8-fa86c4dad572)

**Zaplin Design**
![Screenshot from 2023-09-15 16-34-01](https://github.com/samuelkinyanjui/bookstore_add_reducers/assets/45226848/974fc031-edee-4e60-9bac-497031696c76)